prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -212,6 +212,49 @@ periodics:
         value: prow/oss/config.yaml
       - name: JOB_CONFIG_PATH
         value: prow/prowjobs
+      - name: PROW_INSTANCE_NAME
+        value: oss-prow
+    volumes:
+    - name: github
+      secret:
+        secretName: oauth-token
+- cron: "15 15-23 * * 1-5"  # Run at 7:15-15:15 PST (15:15 UTC) Mon-Fri
+  name: ci-oss-test-infra-autobump-knative-prow
+  cluster: test-infra-trusted
+  decorate: true
+  extra_refs:
+  - org: GoogleCloudPlatform
+    repo: oss-test-infra
+    base_ref: master
+  annotations:
+    testgrid-dashboards: googleoss-test-infra
+    testgrid-tab-name: autobump-knative-prow
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/autobump:v20200622-168a90f1b0
```

Comment:
We should wait to merge until this is updated to a version that includes https://github.com/kubernetes/test-infra/pull/18106

Reply:
Ack

Reply:
`v20200629-32917fc1b6`

Reply:
Thanks, updated on this single job only

Reply:
Also, the PR adds core config is needed before this one, could you please quickly stamp it please? https://github.com/GoogleCloudPlatform/oss-test-infra/pull/407
I have double checked and there is no update since I created the PR